features/fo_new/renewals/renewals_from_user_registrations_page.feature

```diff
@@ -6,11 +6,12 @@ Feature: Registered waste carrier chooses to renew their registration from regis
 
   @smoke
   Scenario: Limited company renews upper tier registration from registrations list
-       Given I have signed in to view my registrations as "another-user@example.com"
-  	     And I choose registration "CBDU213" for renewal
-        When I complete my limited company renewal steps
-         And view my registration on the dashboard
-        Then I will see my registration "CBDU213" has been renewed
+      Given I create a new registration for my "limitedCompany" business as "another-user@example.com"
+        And I have signed in to view my registrations as "another-user@example.com"
+  	    And I choose registration to renew my last registration
```

Comment:
This wording seems a bit confusing. I'd suggest "I choose to renew my last registration from the dashboard" for this and the step definition.

Reply:
Oh!! I didn't notice that! Good catch! :)

Reply:
```suggestion
  	    And I choose to renew my last registration from the dashboard
```